extensions/adobe/experience/offer-management/schema-projection.schema.json

```diff
@@ -0,0 +1,52 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$id": "https://ns.adobe.com/experience/offer-management/schema-projection",
+  "meta:extensible": "true",
+  "type": "object",
+  "title": "Schema Projection",
+  "description":
+    "Schema projection is a way to enable some subset of schema properties for use in queries",
```

Comment:
I don't get what this schema is, what it does, why it's needed. Start with a good description.

Reply:
n/a